pom.xml

```diff
@@ -179,6 +181,281 @@
     </profile>
   </profiles>
 
+  <dependencyManagement>
+    <dependencies>
+
+      <!-- Beam SDK -->
```

Comment:
I'd probably remove these comments, because we aren't consistent (some artifacts have it and others don't), and they aren't particularly informative.

I'd leave comment at the beginning of the testing scope section, and those that describe the exclusions.